test/cli/install/bun-create.test.ts

```diff
@@ -107,6 +107,24 @@ it("should create template from local folder", async () => {
   expect(dirStat.isDirectory()).toBe(true);
 });
 
+it("should not mention cd prompt when created in current directory", async () => {
```

Comment:
@dylan-conway wrote this test, for some reason `test-template` wasn't giving any output so i decided to go with creating a `create-test` directory same as `should create and install github template from`, which worked.

lemme know what changes i should make to this